server/src/main/java/org/apache/uniffle/server/ShuffleDataFlushEvent.java

```diff
@@ -37,7 +38,11 @@ public class ShuffleDataFlushEvent {
   private final int shuffleId;
   private final int startPartition;
   private final int endPartition;
+  /** The memory cost size include encoded length */
   private final long size;
+  /** The data size of this shuffle block */
+  private final long length;
```

Comment:
Could we rename these variables?

Reply:
Yeah, it is a lucky survivor, renamed it.